server/api/emailer.ts

```diff
@@ -149,8 +148,7 @@ class ProductionEmailer implements IEmailer {
             });
             return res.results.total_accepted_recipients === 1;
         } catch (e) {
-            this.log.ERROR('Exception while sending');
-            console.log(e);
+            this.log.ERROR('Exception while sending'); 
```

Comment:
Looks like there's a trailing space on this line. Remove that in a new commit, push it to this branch, and I'll merge this PR in